src/test/java/racingcar/domain/CarTest.java

```diff
@@ -0,0 +1,45 @@
+package racingcar.domain;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import racingcar.domain.Car;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class CarTest {
+    Car car;
+
+    @Test
+    @DisplayName("5글자 초과시 Exception 발생 테스트")
+    public void nameTest(){
+        String name = "sonata";
+        assertThatThrownBy(()-> car = new Car(name)).isInstanceOf(RuntimeException.class);
```

Comment:
참조할 필요 없이 인스턴스만 생성해도 테스트가 가능합니다. ☺️
```suggestion
        String name = "sonata";
        assertThatThrownBy(()-> new Car(name)).isInstanceOf(RuntimeException.class);
```